src/main/java/by/itech/lab/supplier/dto/mapper/UserMapper.java

```diff
@@ -0,0 +1,68 @@
+package by.itech.lab.supplier.dto.mapper;
+
+import by.itech.lab.supplier.domain.User;
+import by.itech.lab.supplier.dto.UserDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper implements BaseMapper<User, UserDto> {
+
+    @Override
+    public UserDto map(User user) {
+        return UserDto.builder()
+                .id(user.getId())
+                .username(user.getUsername())
+                .name(user.getName())
+                .surname(user.getSurname())
+                .email(user.getEmail())
+                .birthday(user.getBirthday())
+                .active(user.isActive())
+                .role(user.getRole())
+                .activeForDeletion(user.isActiveForDeletion())
+                .creatorApplications(user.getCreatorApplications())
+                .updatorApplications(user.getUpdatorApplications())
+                .creatorWayBills(user.getCreatorWayBills())
+                .driverWayBills(user.getDriverWayBills())
+                .address(user.getAddress())
+                .customer(user.getCustomer())
+                .build();
+    }
+
+    public void update(final UserDto from, final User to) {
+        to.setUsername(from.getUsername());
+        to.setName(from.getName());
+        to.setSurname(from.getSurname());
+        to.setEmail(from.getEmail());
```

Comment:
please notice that this method receive two args and in the "to" args there is already specified current values, so my question is: should we allow to change email to activated user?